cypher-shell/src/main/java/org/neo4j/shell/cli/CliArgHelper.java

```diff
@@ -165,6 +164,15 @@ private static ArgumentParser setupParser()
                 .dest("force-non-interactive")
               .action(new StoreTrueArgumentAction());
 
+        parser.addArgument("--width")
```

Comment:
Deeply disagree with this design decision. The program should output a sensible width automatically

Reply:
neo4j-client has a similar option, `:set width=???` (a cli option, allowing width changes at runtime, rather than a command line switch). It also accepts `:set width=auto`, which checks the terminal window size and sets the size based on that (`auto` is the default). Assuming Java supports finding the termwidth, perhaps a width option (with default) is a neat solution here too?

Reply:
well if *I* were designing this, I'd have a single switch: `--wrap[=[false|true|INT]]` where true would be some kind of automatic width (or sensible value).